numpy/core/src/multiarray/datetime.c

```diff
@@ -26,6 +26,8 @@
 #include "_datetime.h"
 #include "datetime_strings.h"
 
+#define DAYS_IN_400_YEARS 146097
```

Comment:
Can you leave this expressed as its constituent parts, as it was on line 157?

Reply:
Yes. Done.